awx/main/scheduler/task_manager.py

```diff
@@ -173,7 +173,7 @@ def process_finished_workflow_jobs(self, workflow_jobs):
                 workflow_job.status = new_status
                 if reason:
                     logger.info(reason)
-                    workflow_job.job_explanation = _("No error handling paths found, marking workflow as failed")
+                    workflow_job.job_explanation = _noop("No error handling paths found, marking workflow as failed")
```

Comment:
Why not just keep the imported name as `gettext_noop`?  `_noop` isn't super clear.